nomos-services/mix/src/lib.rs

```diff
@@ -77,12 +81,41 @@ where
             mut backend,
             network_relay,
         } = self;
+        let mix_config = service_state.settings_reader.get_updated_settings();
 
         let network_relay = network_relay.connect().await?;
         let network_adapter = Network::new(network_relay);
 
-        // TODO: Spawn PersistentTransmission (Tier 1)
-        // TODO: Spawn Processor (Tier 2) and connect it to PersistentTransmission
+        // Spawn Persistent Transmission
+        let (transmission_schedule_sender, transmission_schedule_receiver) =
+            mpsc::unbounded_channel();
+        let (emission_sender, mut emission_receiver) = mpsc::unbounded_channel();
+        tokio::spawn(async move {
+            persistent_transmission(
+                mix_config.persistent_transmission,
+                transmission_schedule_receiver,
+                emission_sender,
+            )
+            .await;
+        });
+
+        // Spawn Message Blend and connect it to Persistent Transmission
+        let (new_message_sender, new_message_receiver) = mpsc::unbounded_channel();
+        let (processor_inbound_sender, processor_inbound_receiver) = mpsc::unbounded_channel();
+        let (fully_unwrapped_message_sender, mut fully_unwrapped_message_receiver) =
+            mpsc::unbounded_channel();
+        tokio::spawn(async move {
+            MessageBlend::new(
+                mix_config.message_blend,
+                new_message_receiver,
+                processor_inbound_receiver,
+                // Connect the outputs of Message Blend to Persistent Transmission
+                transmission_schedule_sender,
+                fully_unwrapped_message_sender,
+            )
+            .run()
+            .await;
+        });
```

Comment:
Third possible confirmation this doesn't look the best way. (But again, I'm not sure if my suggestion would be better for sure).

Reply:
https://github.com/logos-co/nomos-node/pull/844#discussion_r1816233929